2021-03-08 10:45:30,846 INFO    MainThread:1038593 [wandb_setup.py:_flush():69] setting env: {}
2021-03-08 10:45:30,846 INFO    MainThread:1038593 [wandb_setup.py:_flush():69] setting login settings: {}
2021-03-08 10:45:30,846 INFO    MainThread:1038593 [wandb_init.py:_log_setup():334] Logging user logs to /home/naddeok5/FIM/wandb/run-20210308_104530-xbbk5fxk/logs/debug.log
2021-03-08 10:45:30,847 INFO    MainThread:1038593 [wandb_init.py:_log_setup():335] Logging internal logs to /home/naddeok5/FIM/wandb/run-20210308_104530-xbbk5fxk/logs/debug-internal.log
2021-03-08 10:45:30,847 INFO    MainThread:1038593 [wandb_init.py:init():367] calling init triggers
2021-03-08 10:45:30,847 INFO    MainThread:1038593 [wandb_init.py:init():372] wandb.init called with sweep_config: {}
config: {}
2021-03-08 10:45:30,847 INFO    MainThread:1038593 [wandb_init.py:init():416] starting backend
2021-03-08 10:45:30,847 INFO    MainThread:1038593 [backend.py:_multiprocessing_setup():69] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-03-08 10:45:30,874 INFO    MainThread:1038593 [backend.py:ensure_launched():123] starting backend process...
2021-03-08 10:45:30,899 INFO    MainThread:1038593 [backend.py:ensure_launched():127] started backend process with pid: 1038728
2021-03-08 10:45:30,901 INFO    MainThread:1038593 [wandb_init.py:init():421] backend started and connected
2021-03-08 10:45:30,902 INFO    MainThread:1038593 [wandb_init.py:init():461] updated telemetry
2021-03-08 10:45:30,902 INFO    MainThread:1038593 [wandb_init.py:init():480] communicating current version
2021-03-08 10:45:32,201 INFO    MainThread:1038593 [wandb_init.py:init():485] got version response 
2021-03-08 10:45:32,201 INFO    MainThread:1038593 [wandb_init.py:init():493] communicating run to backend with 30 second timeout
2021-03-08 10:45:32,436 INFO    MainThread:1038593 [wandb_init.py:init():518] starting run threads in backend
2021-03-08 10:45:33,712 INFO    MainThread:1038593 [wandb_run.py:_console_start():1476] atexit reg
2021-03-08 10:45:33,712 INFO    MainThread:1038593 [wandb_run.py:_redirect():1339] redirect: SettingsConsole.REDIRECT
2021-03-08 10:45:33,713 INFO    MainThread:1038593 [wandb_run.py:_redirect():1344] Redirecting console.
2021-03-08 10:45:33,713 INFO    MainThread:1038593 [redirect.py:install():213] install start
2021-03-08 10:45:33,713 INFO    MainThread:1038593 [redirect.py:install():228] install stop
2021-03-08 10:45:33,713 INFO    MainThread:1038593 [redirect.py:install():213] install start
2021-03-08 10:45:33,715 INFO    MainThread:1038593 [redirect.py:install():228] install stop
2021-03-08 10:45:33,716 INFO    MainThread:1038593 [wandb_run.py:_redirect():1390] Redirects installed.
2021-03-08 10:45:33,716 INFO    MainThread:1038593 [wandb_init.py:init():542] run started, returning control to user process
2021-03-08 10:45:33,716 INFO    MainThread:1038593 [wandb_run.py:_config_callback():733] config_cb None None {}
2021-03-08 10:45:37,614 INFO    MainThread:1038593 [wandb_run.py:_atexit_cleanup():1446] got exitcode: 1
2021-03-08 10:45:37,615 INFO    MainThread:1038593 [wandb_run.py:_restore():1418] restore
2021-03-08 10:45:37,615 INFO    MainThread:1038593 [redirect.py:uninstall():232] uninstall start
2021-03-08 10:45:37,615 INFO    MainThread:1038593 [redirect.py:_stop():287] _stop: stdout
2021-03-08 10:45:37,616 INFO    stdout    :1038593 [redirect.py:_pipe_relay():129] relay done saw last write: stdout
2021-03-08 10:45:37,616 INFO    MainThread:1038593 [redirect.py:_stop():293] _stop closed: stdout
2021-03-08 10:45:37,616 INFO    stdout    :1038593 [redirect.py:_pipe_relay():145] relay done done: stdout
2021-03-08 10:45:37,616 INFO    MainThread:1038593 [redirect.py:_stop():299] _stop joined: stdout
2021-03-08 10:45:37,616 INFO    MainThread:1038593 [redirect.py:_stop():301] _stop rd closed: stdout
2021-03-08 10:45:37,616 INFO    MainThread:1038593 [redirect.py:uninstall():236] uninstall done
2021-03-08 10:45:37,616 INFO    MainThread:1038593 [redirect.py:uninstall():232] uninstall start
2021-03-08 10:45:37,616 INFO    MainThread:1038593 [redirect.py:_stop():287] _stop: stderr
2021-03-08 10:45:37,616 INFO    stderr    :1038593 [redirect.py:_pipe_relay():129] relay done saw last write: stderr
2021-03-08 10:45:37,616 INFO    MainThread:1038593 [redirect.py:_stop():293] _stop closed: stderr
2021-03-08 10:45:37,616 INFO    stderr    :1038593 [redirect.py:_pipe_relay():145] relay done done: stderr
2021-03-08 10:45:37,617 INFO    MainThread:1038593 [redirect.py:_stop():299] _stop joined: stderr
2021-03-08 10:45:37,617 INFO    MainThread:1038593 [redirect.py:_stop():301] _stop rd closed: stderr
2021-03-08 10:45:37,617 INFO    MainThread:1038593 [redirect.py:uninstall():236] uninstall done
2021-03-08 10:45:37,734 INFO    MainThread:1038593 [wandb_run.py:_wait_for_finish():1568] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 719
  total_bytes: 719
}

2021-03-08 10:45:39,738 INFO    MainThread:1038593 [wandb_run.py:_wait_for_finish():1568] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 9619
  total_bytes: 9619
}

2021-03-08 10:45:41,162 INFO    MainThread:1038593 [wandb_run.py:_show_files():1790] logging synced files
